app/controllers/ParoleParom1ReportController.java

```diff
@@ -113,11 +121,13 @@ public ParoleParom1ReportController(HttpExecutionContext ec,
 
     private Map<String, String> storeCustodyData(Map<String, String> params, String bearerToken, Optional<PrisonerApi.Offender> maybeOffender, Optional<PrisonerCategoryApi.Category> maybeCategory) {
         val nomsNumber = params.get("prisonerDetailsNomisNumber");
+
         params.putAll(maybeOffender
                 .map(offender -> ImmutableMap.<String, String>builder()
                         .put("prisonerDetailsPrisonInstitution", offender.getInstitution().getDescription())
                         .put("prisonerDetailsPrisonNumber", offender.getMostRecentPrisonerNumber())
                         .put("prisonerDetailsPrisonersFullName", offender.displayName())
+                        .put("isYoungAdult", String.valueOf(calculateAge(Optional.ofNullable(offender.getDateOfBirth()).orElse("1970-01-01"), systemUTC()) < 21))
```

Comment:
This ensures that if dateOfBirth is somehow not present, we will treat them as over 21